Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -166,7 +196,7 @@ namespace Hazel {
 		#define HZ_FUNC_SIG __PRETTY_FUNCTION__
 	#elif defined(__DMC__) && (__DMC__ >= 0x810)
 		#define HZ_FUNC_SIG __PRETTY_FUNCTION__
-	#elif defined(__FUNCSIG__)
+	#elif defined(_MSC_VER)
```

Comment:
I thought MVS defined `__FUNCSIG__` at the start of the precompiler? If not, I think we should at least leave it?
```suggestion
	#elif defined(__FUNCSIG__ || _MSC_VER)
```

Reply:
Done.